### ISSUE TITLE
Error en Modelo Ejercicios: Clasificación de texto usando LSTMs con TensorFlow 2.0

### ISSUE THREAD

**Issue by anilupss** [nlp-en-es/nlp-de-cero-a-cien#6]
Me he suscrito al Workshop y me ha parecido genial. He intentado encontrar el código Clasificación de texto usando LSTMs con TensorFlow 2.0 explicado en la sesión 2, pero no lo encuentro. Seguí el código según lo explica @mariagrandury en el video, pero me ha dado un error al momento de crear la red neuronal y de aquí en adelante nada funciona.

model=tf.keras.Sequential([tf.keras.layers.Embedding(vocab_size,embedding_dim),
    tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(embedding_dim)),
    tf.keras.layers.Dense(embedding_dim,activation='relu'),
    tf.keras.layers.Dense(6,activation='softmax')])

Error reportado...
**NotImplementedError:** Cannot convert a symbolic Tensor (bidirectional/forward_lstm/strided_slice:0) to a numpy array. This error may indicate that you're trying to pass a Tensor to a NumPy call, which is not supported

Espero su ayuda urgente.
